azure-devops/build-benchmarks.yml

```diff
@@ -29,18 +29,20 @@ steps:
   displayName: 'Configure the benchmarks for ${{ parameters.compiler }}'
   timeoutInMinutes: 2
   env: { TMP: $(tmpDir), TEMP: $(tmpDir) }
-  # TRANSITION, we currently don't build the benchmarks with Clang for ARM64 or ARM64EC
+  # TRANSITION, we currently only build the benchmarks with Clang for x64
   condition: >
     and(succeeded(), ${{ parameters.buildBenchmarks }},
-      not(and(eq('${{ parameters.compiler }}', 'clang-cl'), startsWith('${{ parameters.targetPlatform }}', 'arm64'))))
+      not(and(eq('${{ parameters.compiler }}', 'clang-cl'),
+        not(startsWith('${{ parameters.targetPlatform }}', 'x64')))))

```

Comment:
`startsWith` was being used to handle both `'arm64'` and `'arm64ec'`. We should use `eq` for `'x64'`.